Source/WTF/wtf/StdLibExtras.h

```diff
@@ -1056,13 +1056,18 @@ std::strong_ordering compareSpans(std::span<T, TExtent> a, std::span<U, UExtent>
     return std::strong_ordering::equal;
 }
 
+template<typename T, typename U>
+concept EqualityComparable = std::is_same_v<std::remove_const_t<T>, std::remove_const_t<U>> || (!std::is_same_v<std::remove_const_t<T>, char8_t> && !std::is_same_v<std::remove_const_t<U>, char8_t>);
+
+template<typename T> concept OneByteType = sizeof(T) == 1 && ((std::is_integral_v<T> && !std::same_as<T, bool>) || std::same_as<T, std::byte>);
```

Comment:
We should format new code consistently with the multiple line style used just above that we discussed in WebKit-dev or Slack. I don't think the name `OneByteType` is ideal given we already have a concept named `ByteType` that is almost identical. The difference between the two seems to be related to `const` and not the number one.

```suggestion
template<typename T>
concept OneByteType = sizeof(T) == 1 && ((std::is_integral_v<T> && !std::same_as<T, bool>) || std::same_as<T, std::byte>);
```